styletesla.css

```diff
@@ -31,6 +34,9 @@
 	align-items: center;
 	flex-direction: column;
 }
+.main__title {
+font: 90px/110px montserratsemibold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5505629)